src/main/java/org/javarosa/test/XFormsElement.java

```diff
@@ -44,7 +44,7 @@ static Map<String, String> parseAttributes(String name) {
         Map<String, String> attributes = new HashMap<>();
         String[] words = name.split(" ");
         for (String word : asList(words).subList(1, words.length)) {
-            String[] parts = word.split("(?<!\\))=(\"|')");
+            String[] parts = word.split("(?<!\\))=(\"|')", 2);
```

Comment:
I ran into this problem when working on a related tests in Collect. Basically `select1Dynamic` did not work with nodeset expressions like `instance('things')/root/item[property='value']` because it would split on the second `=`.

Reply:
😮 That's a useful `split` signature I did not know about!

Reply:
Just discovered it myself!